R/pkg/R/DataFrame.R

```diff
@@ -1903,7 +1903,7 @@ setMethod("except",
 #' }
```

Comment:
@felixcheung  Is it necessary to do that ? Because I notice there's no doc for the default value in other R APIs. Actually user can see the default value in the function signature which is also in the R doc. 
```
## S4 method for signature 'DataFrame,character'
write.df(df, path, source = NULL,
  mode = "error", ...)
```